File: JavaScript/Matriz_2D.js
const matriz2D = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

function recorrido2D(m) {
    let resultado = "";
    for (let fila of m)
        for (let elemento of fila)
            resultado += elemento + " ";
    console.log(resultado);
}

recorrido2D(matriz2D);

File: Java/Matriz_2D.java
public class Matriz_2D {
    public static void main(String[] args) {
        int[][] matriz2D = {
            {1, 2, 3},
            {4, 5, 6},
            {7, 8, 9}
        };

        for (int[] fila : matriz2D)
            for (int elemento : fila)
                System.out.print(elemento + " ");
        System.out.println();
    }
}
